rook/Tester.py

```diff
@@ -173,7 +173,12 @@ def get_valid_params():
     params = _ValidParameters()
     params.add_required_param('type', 'The type of this differ')
     params.add_required_param('output', 'Output files to check')
-    params.add_param('gold_files', '', 'Gold filenames')
+    params.add_param('windows_gold', '', 'Paths to Windows specific gold files,'+
+                     ' relative to test directory')
+    params.add_param('mac_gold', '', 'Paths to Mac specific gold files, relative to test directory')
+    params.add_param('linux_gold', '', 'Paths to Linux specific gold files,'+
+                     ' relative to test directory')
+    params.add_param('gold_files', '', 'Paths to gold files, relative to test directory')
```

Comment:
I guess our assumption here is to use the path relative to gold directory

Reply:
Currently the path is relative to the test directory, not the gold directory. I initially wanted to have things relative to the gold directory, but some tests generate some data for a ROM, build the ROM, and then check that the ROM gives the same data generated previously. It would not be too difficult to change it to assume the path is relative to the gold directory.